test/input/TestSampleSheet/SampleSheet-custom-1_macos9-endings.txt

```diff
@@ -0,0 +1 @@
+sample	barcode_1	barcode_2	library_id_per_sample
NIHCC_05	CGTACTAG	TAGATCGC	AEL2
NIHCC_06	AGGCAGAA	TAGATCGC	AEL2
NIHCC_07	TCCTGAGC	TAGATCGC	AEL2
NIHCC_08	TAGGCATG	TAGATCGC	AEL2
NIHCC_09	GCTACGCT	TAGATCGC	AEL2
NIHCC_10	CGAGGCTG	TAGATCGC	AEL2
NIHCC_11	AAGAGGCA	TAGATCGC	AEL2
NIHCC_12	GTAGAGGA	TAGATCGC	AEL2
NIHCC_13	CGTACTAG	CTCTCTAT	AEL2
NIHCC_14	AGGCAGAA	CTCTCTAT	AEL2
NIHCC_15	TCCTGAGC	CTCTCTAT	AEL2
NIHCC_16	TAGGCATG	CTCTCTAT	AEL2
NIHCC_17	GCTACGCT	CTCTCTAT	AEL2
NIHCC_18	CGAGGCTG	CTCTCTAT	AEL2
NIHCC_19	AAGAGGCA	CTCTCTAT	AEL2
NIHCC_20	GTAGAGGA	CTCTCTAT	AEL2
NIHCC_21	CGTACTAG	TATCCTCT	AEL2
NIHCC_22	AGGCAGAA	TATCCTCT	AEL2
NIHCC_23	TCCTGAGC	TATCCTCT	AEL2
NIHCC_24	TAGGCATG	TATCCTCT	AEL2
NIHCC_25	GCTACGCT	TATCCTCT	AEL2
NIHCC_26	CGAGGCTG	TATCCTCT	AEL2
NIHCC_27	AAGAGGCA	TATCCTCT	AEL2
NIHCC_28	GTAGAGGA	TATCCTCT	AEL2

```

Comment:
I'm always slightly worried about git auto-converting `.txt` files, but it does look like this came through properly.